subxt/src/backend/legacy/rpc_methods.rs

```diff
@@ -268,6 +323,49 @@ impl<T: Config> LegacyRpcMethods<T> {
         Ok(subscription)
     }
 
+    /// Insert a key into the keystore.
+    pub async fn author_insert_key(
+        &self,
+        key_type: String,
+        suri: String,
+        public: Vec<u8>,
+    ) -> Result<(), Error> {
+        let params = rpc_params![key_type, suri, Bytes(public)];
+        self.client.request("author_insertKey", params).await?;
+        Ok(())
+    }
+
+    /// Generate new session keys and returns the corresponding public keys.
+    pub async fn author_rotate_keys(&self) -> Result<Vec<u8>, Error> {
+        let bytes: Bytes = self
+            .client
+            .request("author_rotateKeys", rpc_params![])
```

Comment:
Hmm, are we adding those back to the legacy API since they might be useful for validators? 🤔